frontend/components/burn-fraction-button.tsx

```diff
@@ -27,13 +29,17 @@ export const BurnFractionButton = ({
    * Related to https://github.com/Network-Goods/hypercerts-protocol/issues/80
    */
 
-  const handleClick = () => {
-    write(fractionId);
+  const handleClick = async () => {
+    const writeable = await checkWriteable();
```

Comment:
I'm a little confused why export both `checkWriteable` and `writeable`, if we're going to go ahead and call it ourselves here again anyway?

Shouldn't we just be able to use `writeable` from the hook?

Reply:
The difference is that `writeable` does all the checks silently and logs the issues (so the button is disabled but your not prompted with toasts). On this line you do get toasts if any of the checks fails and on what. 

Agree that the checks might be a little redundant, the thought was to be extra sure. Want me to take one out?